home/import_content_pages.py

```diff
@@ -339,91 +342,110 @@ def add_variation_to_shadow_content_page_from_row(
             )
         )
 
-    def add_message_to_shadow_content_page_from_row(
-        self, row: "ContentRow", locale: Locale
-    ) -> None:
+    def _get_shadow_page(self, slug: str, locale: Locale) -> Page:
+
         try:
-            page = self.shadow_pages[(row.slug, locale)]
+            return self.shadow_pages[(slug, locale)]
         except KeyError:
             raise ImportException(
-                f"This is a message for page with slug '{row.slug}' and locale '{locale}', but no such page exists"
+                f"This is a message for page with slug '{slug}' and locale '{locale}', but no such page exists"
             )
-        if row.is_whatsapp_message:
-            page.enable_whatsapp = True
-            buttons = []
-            for index, button in enumerate(row.buttons):
-                if button["type"] == "next_message":
-                    buttons.append(
+
+    def _get_form(
+        self, slug: str, locale: Locale, title: str, page_slug: str, field: str
+    ) -> Assessment:
+        try:
+            return Assessment.objects.get(slug=slug, locale=locale)
+        except Assessment.DoesNotExist:
+            raise ImportException(
+                f"No form found with slug '{slug}' and locale '{locale}' for go_to_form {field} '{title}' on page '{page_slug}'"
+            )
+
+    def _create_interactive_items(
+        self,
+        row_field: list[dict[str, Any]],
+        page: Page,
+        slug: str,
+        locale: Locale,
+        field_name: str,
+    ) -> list[dict[str, Any]]:
+        messages: list[dict[str, Any]] = []
+        for index, content_page_field in enumerate(row_field):
```

Comment:
These aren't messages or fields, so maybe:
- `item_type` for the parameter (which is either `"button"` or `"list item"`)
- `items` for the list we're collecting into (instead of `messages`)
- `item` for the dict we're processing (instead of `field`)

would be better names?